### PR TITLE
Adds Tabulate

### PR DESCRIPTION
**Pull request by marimeireles**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

Please let me know if you want me to rebase so you have a clean history  of commits!

### CODE REVIEW

**Comment by wolfv** [comment]
hey @marimeireles I think you just need to change the name to `cpp-tabulate` or something similar, because tabulate is also a Python library :)

**Comment by wolfv** [comment]
I think the `skip: true` needs to go aways, otherwise _all_ builds are skipped :)

**Comment by wolfv** [comment]
also, the name of the license file in the tabulate repository is `LICENSE`  (not LICENSE.txt)

https://github.com/p-ranav/tabulate/blob/master/LICENSE

**Comment by marimeireles** [comment]
Oh! sorry I missed that! 
And thanks for the reviews!

**Comment by marimeireles** [comment]
Hey @p-ranav I created a conda package for Tabulate. Would you like to be included as one of the maintainers for it? :)
Thanks for the awesome work!

**Comment by p-ranav** [comment]
@marimeireles Sure :)

**Comment by marimeireles** [comment]
@SylvainCorlay could you please check this one out when you have the time? :)
Thanks!

File: recipes/tabulate/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```
Comment (chrisburr) [inline recipes/tabulate/meta.yaml]: Can you delete these 5 lines

**Comment by marimeireles** [comment]
@conda-forge/help-c-cpp

**Comment by marimeireles** [comment]
@wolfv could you merge eventually? =)

**Pull request merged**